src/sentry/scmode.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
```

Comment:
Could this file be named as `server_component_mode.py`? 

The long form name makes it easier to grep.